editors/emacs/lambdapi.el

```diff
@@ -14,10 +14,18 @@
 ;; Keywords
 (defconst lambdapi-font-lock-keywords
   (list (cons (concat "\\<" (regexp-opt '(
-    "require" "open" "as" "let" "in" "symbol" "const" "injective" "rule" "and"
-    "definition" "theorem" "assert" "assertnot" "set" "proof" "qed" "admit"
-    "abort" "focus" "print" "proofterm" "refine" "apply" "simpl" "assume"
-    "rewrite" "refl" "sym" "type" "compute"
+    "require" "open" "as" "let" "in"
```

Comment:
It would be good to put all these strings in lexicographic order. (There is a command in Emacs for that after @gabrielhdt .)

Reply:
fixed in 24c13d9